public/less/_repository.less

```diff
@@ -1836,15 +1835,40 @@ tbody.commit-list {
 }
 
 #topic_edit {
-    margin-top:5px;
+    margin-top: 5px;
 }
 
-#repo-topic {
+#repo-topics {
     margin-top: 5px;
 }
 
+.repo-topic {
+    cursor: pointer;
+}
+
 .new-dependency-drop-list {
-	@media only screen and (max-width: 768px) {
-		width: 100%;
-	}
+	  @media only screen and (max-width: 768px) {
```

Comment:
This formatting looks a little off

Reply:
It isn't really off, just made it consistent to 4-space indent. But now that I look at it, I'm not sure this will even work, you usually wrap selectors inside a media query, not the other way around, fixing that.